bin/post-link-live-branch.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+### Expected binaries
+# - jq
+# - curl
+### Expected en vars
+# - GH_TOKEN
+
+function get() {
+	url="${1}"
+	curl --fail --silent --show-error --location \
+		--header "Accept: application/vnd.github.v3+json" \
+		--header "Authorization: token ${GH_TOKEN}" \
+		"${url}"
+}
+
+function post() {
+	url="${1}"
+	data="${2}"
+	curl --fail --silent --show-error --location \
+		--header "Accept: application/vnd.github.v3+json" \
+		--header "Authorization: token ${GH_TOKEN}" \
+		-X POST "${url}" --data "${data}"
+}
+
+# Sanitize parameters
+branch="${1:-}"
+link="${2:-}"
+if [[ -z "$branch" ]]; then
+	echo "Usage: ${0} <branch-name> [link]"
+	exit 1
+fi
+
+# Compose messages
+WATERMARK="<!--calypso.live-watermark-->"
+GENERATING=$(jq -Rs . <<- EOF
+	${WATERMARK}
+	Link to live branch is being generated...
+	Please wait a few minutes and refresh this page.
+EOF
+)
+LINK=$(jq -Rs . <<- EOF
+	${WATERMARK}
+	Link to live branch: ${link}
+EOF
+)
+if [[ -z "${link}" ]]; then
+	message="${GENERATING}"
+else
+	message="${LINK}"
+fi
+
+echo "Getting PRs associated with branch '${branch}'..."
+prNumber=$(get "https://api.github.com/repos/Automattic/wp-calypso/pulls?head=Automattic:${branch}" | jq '.[].number')
```

Comment:
@scinos What do you think about creating a separate script for posting a one-time comment to the GitHub PR? I would also find this very useful for ETK.

Reply:
I'm not sure what you mean. Which parts do you think they can be made more general? Pass the messages as arguments?

Reply:
Thanks a ton, that's exactly what I meant :)